packages/client/hmi-client/src/workflow/ops/simulate-ciemss/tera-simulate-ciemss.vue

```diff
@@ -198,11 +231,19 @@ const selectedRunId = computed(
 	() => props.node.outputs.find((o) => o.id === selectedOutputId.value)?.value?.[0]
 );
 
+const parentSize = ref({ width: 0, height: 270 }); // Set the initial height or any default height
+// Calculate the parent container's width
```

Comment:
Do not use this parentSize method for chart. 
I doubt we want to change how Dan just updated `chartSize` in main